doc/nrf/libraries/others/profiler.rst

```diff
@@ -13,129 +13,201 @@ The output is provided using RTT and can be visualized in a custom Python backen
 
 See the :ref:`profiler_sample` sample for an example of how to use the Profiler.
 
+.. _profiler_configuration:
+
 Configuration
 *************
 
-Apart from standard configuration parameters, there is one required setting:
+The Profiler can be configured to profile either custom events or Event Manager events.
```

Comment:
Profiler can profile both too (after it was upgraded by Aleksander Strzeboński)

Reply:
I would not distinguish these as 2 alternatives, since Event Manager events are converted to profiler events.